samples/sampler/stories/qa/Panels.js

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable react/jsx-no-bind */
+import {handle, forward} from '@enact/core/handle';
+import kind from '@enact/core/kind';
+import {clamp} from '@enact/core/util';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select} from '@enact/storybook-utils/addons/knobs';
+import Routable, {Route, Linkable} from '@enact/ui/Routable';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import Header from '@enact/agate/Header';
+import Icon from '@enact/agate/Icon';
+import Item from '@enact/agate/Item';
+import {Panels, Panel, BreadcrumbPanels} from '@enact/agate/Panels';
+
+Panels.displayName = 'Panels';
+const Config = mergeComponentMetadata('Panels', Panels);
+
+const BasicPanels = (props) => {
+	const [index, setIndex] = React.useState(0);
+	const goNext = () => setIndex(clamp(0, 2, index + 1));
+	const goPrevious = () => setIndex(clamp(0, 2, index - 1));
+
+	return (
+		<Panels
+			index={index}
+			{...props}
+		>
+			<Panel>
+				<Header title="First Panel" />
+				<Item onClick={goNext}>Next</Item>
+			</Panel>
+			<Panel>
+				<Header title="Second Panel" />
+				<Item onClick={goPrevious}>Previous</Item>
+				<Item onClick={goNext}>Next</Item>
+			</Panel>
+			<Panel>
+				<Header title="Third Panel" />
+				<Item onClick={goPrevious}>Previous</Item>
+			</Panel>
+		</Panels>
+	);
+};
+
+const MenuItemBase = kind({
+	name: 'MenuItem',
+
+	propTypes: {
+		css: PropTypes.object,
+		icon: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+		path: PropTypes.string
+	},
+
+	defaultProps: {
+		disabled: false
+	},
+
+	styles: {
+		className: 'menuItem'
+	},
```

Comment:
Does is need a className?

Reply:
I removed it along with some other unnecessary code